src/components/ItemList/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import { updateItemDate } from '../Utils/firestore';
+import { differenceInHours } from 'date-fns';
+
+function ItemList({ itemName, docId, lastPurchasedDate }) {
+  const [isChecked, setIsChecked] = useState(false);
+  const [isDisabled, setIsDisabled] = useState(false);
+  const [token] = useState(() => window.localStorage.getItem('tcl18-token'));
+  const toggle = () => setIsChecked(!isChecked);
+
+  const editProduct = (id) => {
```

Comment:
It's only a suggestion, editProduct could have `token` as parameter. I feel something magic with `token` in line 12 😄

Reply:
It´s not necessary due to the variable token it's in the scope of the component.